spec/solver_spec.rb

```diff
@@ -0,0 +1,59 @@
+require_relative '../solver'
+
+describe Solver do
+  let(:solver) { Solver.new }
+
```

Comment:
- [ ] As per the requirements, kindly remove and add the solver.rb as a proof of TDD. You can see below that it is a requirement:
![image](https://user-images.githubusercontent.com/91486791/233800753-1cc6362f-a573-43a4-83fe-8a90eb70b96a.png)